recipes/petsc/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+export PETSC_DIR=$SRC_DIR
+export PETSC_ARCH=arch-conda-c-opt
+
+$PYTHON ./configure \
+  --with-fc=0 \
+  --with-debugging=0 \
+  --COPTFLAGS=-O3 \
+  --CXXOPTFLAGS=-O3 \
+  --LIBS=-Wl,-rpath,$PREFIX/lib \
+  --with-blas-lapack-dir=$PREFIX \
+  --with-cmake=0 \
+  --with-hwloc=0 \
+  --with-ssl=0 \
+  --with-x=0 \
+  --prefix=$PREFIX
+
+sedinplace() { [[ $(uname) == Darwin ]] && sed -i "" $@ || sed -i"" $@; }
+for path in $PETSC_DIR $PREFIX; do
+    sedinplace s%$path%\${PETSC_DIR}%g $PETSC_ARCH/include/petsc*.h
+done
+
+make
+make install
+
+if [[ $(uname) == Darwin ]];
+then
+    library=$PREFIX/lib/lib$PKG_NAME.dylib
+else
+    library=$PREFIX/lib/lib$PKG_NAME.so
+fi
+$PYTHON $RECIPE_DIR/replace-binary.py $(dirname $SRC_DIR) "" $library
```

Comment:
I'm not sure this is needed either because conda-build also fixes binary prefixes.